gdc_ng_models/models/release_log.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Integer, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+DATA_TYPE_ENUM = Enum("ssm", "cnv", "case", name="release_log_data_type")
+
+
+class ReleaseLog(Base):
+
+    __tablename__ = "release_log"
+    __table_args__ = (
+        Index("release_log_project_id_idx", "program_name", "project_code"),
+    )
+
+    def __repr__(self):
+        return "<ReleaseLog(project_id={}-{}, release_number='{}', data_type={}, open='{}')>".format(
```

Comment:
Why are some values bare and other are quotes?   Like ``project_id={}-{}`` vs ``release_number='{}'``?